src/renderer/composables/colors.js

```diff
@@ -74,5 +74,19 @@ export function useColorTranslations() {
     t('Settings.Theme Settings.Main Color Theme.Solarized Blue'),
     t('Settings.Theme Settings.Main Color Theme.Solarized Cyan'),
     t('Settings.Theme Settings.Main Color Theme.Solarized Green'),
+    t('Settings.Theme Settings.Main Color Theme.Everforest Dark Red'),
```

Comment:
Sorting must be applied in `ThemeSettings.vue` and `colors.js` files

Reply:
Should I sort the colors via their alphabetical name? (e.g. start with Aqua, Blue, ...) or just keep them as they are?

Reply:
Keep it as is. We havent applied any kind of color sorting for all the other themes